integrated_channels/blackboard/client.py

```diff
@@ -464,3 +577,50 @@ def _get_or_create_integrated_grade_column(self, bb_course_id):
                 )
 
         return grade_column_id
+
+    def create_integration_content_for_course(self, bb_course_id, content_metadata):
```

Comment:
can you double check but I thin kthe proper name for this var is channel_metadata_item? at least that is what we used in other methods. I think content_metadata may refer to the pre-exporter dataset while channel_metadata (and channel_metadata_item) to the post-exporter.. but I would check if this is 100% true. if so let's rename this var?

Reply:
I used `content metadata` because our base methods for transmitting are named `update_content_metadata` and `create_content_metadata`. As for meaning, I don't see a major difference in the two labels. However, I like consistency and this is the obvious outlier, so willing to change!